src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -41,12 +44,13 @@ public RepoConfigCsvParser(Path csvFilePath) throws IOException {
      */
     @Override
     protected void processLine(List<RepoConfiguration> results, String[] elements) throws InvalidLocationException {
-        String location = getValueInElement(elements, LOCATION_POSITION);
+        Location location = new Location(getValueInElement(elements, LOCATION_POSITION));
         String branch = getValueInElement(elements, BRANCH_POSITION, RepoConfiguration.DEFAULT_BRANCH);
-        List<String> formats = getManyValueInElement(elements, FILE_FORMATS_POSITION);
+        List<Format> formats = Format.convertStringsToFormats(getManyValueInElement(elements, FILE_FORMATS_POSITION));
         List<String> ignoreGlobList = getManyValueInElement(elements, IGNORE_GLOB_LIST_POSITION);
         String ignoreStandaloneConfig = getValueInElement(elements, IGNORE_STANDALONE_CONFIG_POSITION);
-        List<String> ignoreCommitList = getManyValueInElement(elements, IGNORE_COMMIT_LIST_CONFIG_POSITION);
+        List<Commit> ignoreCommitList = Commit.convertStringsToCommits(
```

Comment:
`InvalidArgumentException` may be thrown by `Format#convertStringsToFormats()` and `Commit#convertStringsToCommits()`, if given inputs are invalid. 

Also, please do a failing test to check it is handled 😃.

Reply:
Should it throw a `ParseException` when an `IllegalArgumentException` is caught or should it just log the error and use an empty list instead? Since as opposed to not being able to proceed without a valid location, the parsing can continue even without formats/ignore commits.

Reply:
Since we been using `IllegalArgumentException` before, let's just keep it as it is for now. We can fix it in another PR unless the exception is not handled anywhere.

Reply:
IllegalArgumentException is not handled anywhere currently (the application stops and the stack trace is shown to the user, 
`Exception in thread "main" java.lang.IllegalArgumentException: A
        at reposense.parser.RepoConfigCsvParser.processLine(RepoConfigCsvParser.java:49)
        at reposense.parser.CsvParser.parse(CsvParser.java:60)
        at reposense.RepoSense.getRepoConfigurations(RepoSense.java:68)
        at reposense.RepoSense.main(RepoSense.java:42)`

I'll just catch it in the CsvParser#parse and handle it similarly to ParseException for now. (Show a message telling the user it is invalid and stop there).